ruff.toml

```diff
@@ -0,0 +1,5 @@
+line-length = 120
```

Comment:
Consider also adding `target-version = "py310"` and keep it in sync with our earliest supported version